### DISCUSSION TITLE
Web app working well locally but giving me error when deployed on Vercel

### DISCUSSION THREAD

**Discussion by xOluwaseyi** [vercel/next.js#42496] [Help] [answered]
I just audio player project. It worked completely fine locally but when I deployed it on Vercel, it displayed this error on first render:

"Application error: a client-side exception has occurred (see the browser console for more information)."

I followed this [discussion](https://github.com/vercel/next.js/discussions/36121) but it didn't work.

Live url: https://musica-webapp.vercel.app/
GitHub repo: https://github.com/xOluwaseyi/musica

![musica-productionbug.png](https://user-images.githubusercontent.com/75592533/200032774-09e62b67-95d7-4739-b97f-ec6457178e7b.png)

**Comment by wwwdepot** [comment, marked as answer]
Local storage items can be null. You need to add a condition check

https://github.com/xOluwaseyi/musica/blob/e6da18c7bd8ce0572fad4b3cb3a973be50f00fb3/context/index.js#L247-L253

**Reply by xOluwaseyi** [reply]
> Local storage items can be null. You need to add a condition check
> 
> https://github.com/xOluwaseyi/musica/blob/e6da18c7bd8ce0572fad4b3cb3a973be50f00fb3/context/index.js#L247-L253
> 

It seems that was indeed the problem as I haven't gotten any error again since then, thank you. I'll be marking this as the answer 😄
